package.json

```diff
@@ -174,6 +174,11 @@
                 "description": "Environment variables passed to the program.",
                 "default": {}
               },
+              "bind": {
+                "type": "string",
+                "description": "Address to bind to when listening for XDebug",
+                "default": "::"
```

Comment:
Ideally this would default to null if possible, but not familiar with VS Code plugins to know if that is or not.